.github/workflows/ci.yml

```diff
@@ -1,8 +1,12 @@
 ---
 name: CI
 on:
-  - pull_request
-  - push
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Could we also use only "push" (for all branches)? Then we'd cover new PRs (which have been created from a recently pushed branch) as well as pushed to existing PRs and merges to master.